waterlog/src/actions/segmentLeaksActions.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_SEGMENTS_LEAKS } from "./types";
+
+export const fetchSegmentsLeaks = () => dispatch => {
+  console.log("please");
+  fetch('https://localhost:44382/api/segmentleaks')//Change to use either localhost/server
```

Comment:
Please add some error handling on the fetch to prevent the app crashing if the http call fails for some reason. For more information on how to do this you can check: https://www.tjvantoll.com/2015/09/13/fetch-and-errors/

I would suggest in the part where you catch the error to dispatch an error action and in the reducer you can reset the state to a default state